sass/_Variables.scss

```diff
@@ -71,19 +71,19 @@ $default-border: 1px solid $color-light-grey;
 }
 
 @mixin iconWithHoverState($iconBase, $float:'left', $inactive : 'coveo-disabled') {
-  $iconActive: #{$iconBase} + '-active';
+  $iconActive: '#{$iconBase}-active';
   .coveo-icon {
-    @extend #{$iconBase};
+    @extend #{$iconBase} !optional;
     float: #{$float};
   }
   &:hover {
     .coveo-icon {
-      @extend #{$iconActive};
+      @extend #{$iconActive} !optional;
```

Comment:
Yeah for some reason sass can't resolve sprites classes when passed to this mixin... I will investigate more, I'm curious to know why.